resources/web/docs_js/components/console_widget.jsx

```diff
@@ -62,6 +62,56 @@ export class _ConsoleForm extends Component {
   }
 }
 
+export class _TryConsoleSelector extends Component {
+  componentWillMount() {
+    const defaultVals = omit(['langStrings', 'saveSettings', 'url_label', 'setting'], this.props);
+    this.setState(defaultVals);
+  }
+
+
+  render(props) {
+    const handleConfigureClick = (e) => {
+      e.preventDefault();
+      props.settingsModalAction();
+    }
+
+    return (
+      <div className="try_console_selector">
+        <h4>Try in Console</h4>
+        <p>We were unable to detect a running console server.</p>
+        <p>
+          <a
+            id="try_console_selector_try_cloud_button"
+            className="button btn-primary btn-small"
+            href="https://cloud.elastic.co/registration"
+            target="_blank"
+          >
+            Sign up for Elastic Cloud trial
```

Comment:
```suggestion
            Start a free trial of the Elasticsearch Service
```
Is this too long? I'm looking at our [terminology guide](https://docs.google.com/document/d/1GD50O8BgOAAp9OhJTdouISEfrpu5VRww4lvX4pom9QY/edit).

Reply:
We are all over the board here. We never say `Elasticsearch Service`, but instead use `Cloud`. 

Let's use:

`Start a free Cloud trial`

Reply:
From [the terminology guide](https://docs.google.com/document/d/1GD50O8BgOAAp9OhJTdouISEfrpu5VRww4lvX4pom9QY/edit):

> Preferred terms are “on Elasticsearch Service” or “on Elastic Cloud” if there is space. If not “on cloud” is OK. 

Should we use Elastic Cloud instead?

Reply:
> We never say Elasticsearch Service, but instead use Cloud.

Where is this guidance from btw? This goes against the terminology recommendations that I know.

Reply:
`Start a free Elastic Cloud trial` works for me. 

I'm working from the messaging on https://www.elastic.co/cloud.